scripts/start-setupVanillaTweaks

```diff
@@ -81,4 +83,20 @@ if [[ "$PACKS" ]] && [[ "$VT_VERSION" ]]; then
   fi
 fi
 
+# Download and move crafting tweaks
+if [[ "$CRAFTING_TWEAKS" ]] && [[ "$VT_VERSION" ]]; then
+  VT_ZIPDATA_URL=https://vanillatweaks.net/assets/server/zipcraftingtweaks.php
+    DOWNLOAD_URL=$(curl -X POST -F "packs=${CRAFTING_TWEAKS}" -F "version=${VT_VERSION}" $VT_ZIPDATA_URL | jq -r '.link')
+  if [ ! "$DOWNLOAD_URL" ]; then
+    log "ERROR: unable to retrieve crafting tweak DOWNLOAD_URL from vanillatweaks.net!"
+    exit 2
+  fi
+
+  mkdir -p "$out_dir"
+  if ! get -o $out_dir/crafting_tweaks.zip "https://vanillatweaks.net${DOWNLOAD_URL}"; then
+    log "ERROR: failed to download crafting tweaks from ${DOWNLOAD_URL}"
+    exit 2
+  fi
```

Comment:
Looks like cleanup is missing. Probably can be taken out of line 78-83 and made into its own follow up code block.

Reply:
No cleanup required, you just move the zip into the datapacks